.github/workflows/ci.yml

```
@@ -17,7 +17,8 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci-elasticsearch.yml@v4.2.0
+    uses: fastify/workflows/.github/workflows/plugins-ci-elasticsearch.yml@v5.0.1
```

```suggestion
    uses: fastify/workflows/.github/workflows/plugins-ci-elasticsearch.yml@v5
```

I think `v5` should work now since @Fdawgs released using the workflow